src/styles.css

```diff
@@ -154,12 +154,15 @@ nav {
 .ingredients-cost-section {
   display: flex;
   flex-direction: column;
-  width: 18%;
   margin: 0px 0px 30px 30px;
   border: 1px solid #1B3231;
   padding: 0px 15px 0px 15px;
 }
 
+.left-wrapper {
```

Comment:
Fixing the yellow 'notifier' so it was more in line with the rest of our CSS formatting in how it looks.